bandit/plugins/ast_overflow_exec.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding:utf-8 -*-
+#
+# Copyright 2018 Hewlett-Packard Development Company, L.P.
```

Comment:
Don't need HP in the copyright (unless you work there and did this under HP's time and salary).